addons/mail/static/src/messaging/component/follower_list_menu/follower_list_menu_tests.js

```diff
@@ -124,6 +125,172 @@ QUnit.test('base rendering editable', async function (assert) {
     );
 });
 
+QUnit.test('base rendering with multiple partner followers', async function (assert) {
+    assert.expect(7);
+
+    await this.start({
+        async mockRPC(route, args) {
+            if (
+                route === '/web/dataset/call_kw/res.partner/read' &&
+                args.args[1][0] === 'message_follower_ids'
+            ) {
+                return [{
+                    message_follower_ids: [1, 2],
+                }];
+            }
```

Comment:
Can't be done with `mock_server:_mockRead()`?

Reply:
Don't know, don't understand how mockread works at first sight

Reply:
at a glance, it just returns values of provided record fields, with special sauce for some kinds of fields like x2many relations. It relies on data records (`data[<model>].records`) and field definitions (`data[<model>].fields`)